.env.sample

```diff
@@ -22,6 +22,11 @@
 # The URL for an external help resource; defaults to an empty string
 # HELP_URL=https://github.com/tl-its-umich-edu/canvas-app-explorer
 
+# (optional) The token for accessing the /watchman URL; leaving it blank means the route is unprotected
+# DJANGO_WATCHMAN_TOKENS=
+# (optional) The name of the header or parameter used for passing the above token
+# DJANGO_WATCHMAN_TOKEN_NAME=
```

Comment:
I was successfully able to check the watchman endpoint receiving the DB, cache status using a query parameter via GET call from the browser.  When I removed the token from the ConfigsMap the endpoint became optional and it worked very well.